content/extraProblems.json

```diff
@@ -591,6 +591,19 @@
       "solutionMetadata": {
         "kind": "internal"
       }
+    },
```

Comment:
if this already has an internal solution, it's already in extraProblems, right?

Reply:
> do you have java code? (this is for cpi video classes, right?)

yes its for cpi classes. i don't have java code yet but ill add it tomorrow.

Reply:
> if this already has an internal solution, it's already in extraProblems, right?

:( oops I didn't see that it already exists in the guide.